specification/schemas/Email.json

```diff
@@ -1,5 +1,6 @@
 {
   "type": "string",
+  "format": "email",
```

Comment:
If we have the pattern what value does the format bring?

Reply:
Nothing, just for clarification. Maybe they eventually update the json-api package to mention the `format` when the `pattern` does not match.